### PR TITLE
Rewrite connectors and forwarders tests to run clients on cluster

### PR DESCRIPTION
**Pull request by lulf**

### CODE REVIEW

**Comment by lulf** [comment]
@enmasse-ci run tests profile=isolated testcase=io.enmasse.systemtest.isolated.bridging.**

File: systemtests/src/main/java/io/enmasse/systemtest/utils/AddressSpaceUtils.java [hunk 1]
```diff
@@ -219,6 +220,30 @@ public static void waitForAddressSpaceDeleted(AddressSpace addressSpace) throws
         }
     }
 
+    public static Endpoint getServiceEndpointByName(AddressSpace addressSpace, String endpoint, String protocol) {
```
Comment (kornys) [inline systemtests/src/main/java/io/enmasse/systemtest/utils/AddressSpaceUtils.java:223]: @lulf I have same method in my smoke messaging branch on cluster so I don't think we need 2 same methods

**Comment by lulf** [comment]
@enmasse-ci run tests profile=isolated testcase=io.enmasse.systemtest.isolated.bridging.**

**Comment by lulf** [comment]
@enmasse-ci run tests profile=isolated testcase=io.enmasse.systemtest.isolated.bridging.**

**Comment by lulf** [comment]
@enmasse-ci run tests profile=isolated testcase=ForwardersTest#testForwardToFullQueue

**Comment by lulf** [comment]
@kornys Ok to merge?

**Pull request merged**